test/unit/test_smdataparallel.py

```diff
@@ -274,7 +293,7 @@ def test_smdataparallel_run_single_node_python(
         async_shell.assert_called_once()
         async_gather.assert_called_once()
         assert process == async_shell.return_value
-        path_exists.assert_called_with("/usr/sbin/sshd")
+        path_exists.assert_any_call("/usr/sbin/sshd")
```

Comment:
why was this changed?

Reply:
`assert_called_with` checks the latest call to that method. 
Since we added some path exists checks for SMDDP which happen after the sshd checks, this assertion fails. 
Hence, updated it to use `assert_any_call` instead.